tsconfig-strict.json

```diff
@@ -394,6 +394,7 @@
     "core/templates/pages/exploration-player-page/learner-experience/continue-button.component.ts",
     "core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.ts",
     "core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.spec.ts",
+    "core/templates/pages/exploration-player-page/modals/display-hint-modal.component.ts",
```

Comment:
Can you please also type the spec file for this file.

Reply:
Yaa @vojtechjelinek. I am working on it. Will add a commit once it's done. Thanks.

Reply:
Done!